docs/RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# Release process and check offs (client only)
+______
+since there is no server component at this time
+
+## Checkoffs
+
+## Product checks
+1. [ ]  fidelity checks with mockups (design) \*
```

Comment:
Ideally we'll start providing designs for both "general mobile" and "general tablet" for upcoming versions, but I imagine some of this will still require some collaboration with Eng, as you laid out in the statement you added below-

Reply:
I've updated the statement below per @davidkaneda's feedback on slack. I am going to resolve this now - we can reopen if @viviancromwell disagrees

Reply:
Addressed by adding link to https://github.com/WorldHealthOrganization/app/issues/243